spec/spec_helper.rb

```diff
@@ -0,0 +1,109 @@
+require 'omniauth'
+
+OmniAuth.config.test_mode = true
+
+omniauth_hash = OmniAuth::AuthHash.new({
+  'provider': 'auth_provider',
+  'uid':      '00001',
+  'info': {
+    'name':       'Hello World',
+    'email':      'hello_world@example.com',
+    'first_name': 'Hello',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.